packages/gp2-components/src/templates/WorkingGroupOverview.tsx

```diff
@@ -0,0 +1,36 @@
+import { WorkingGroupResponse } from '@asap-hub/model/src/gp2';
+import { TeamMembersSection } from '@asap-hub/react-components';
+import { css } from '@emotion/react';
+
+const styles = css({
+  display: 'grid',
+  gridRowGap: `${36 / 17}em`,
+});
+
+type WorkingGroupOverviewProps = Pick<WorkingGroupResponse, 'members'>;
+
+const WorkingGroupOverview: React.FC<WorkingGroupOverviewProps> = ({
+  members,
+}) => (
+  <div css={styles}>
+    {members.length ? (
```

Comment:
I would have done it like 
```
!!members.length && <section> ... 
```

Reply:
im only copying other engineer's approach https://github.com/yldio/asap-hub/blob/ffa7f88b22d6b919f90a90ca8193e665c5527f0f/packages/react-components/src/templates/TeamProfileAbout.tsx#L50